consensus/block_validation.go

```diff
@@ -71,6 +71,11 @@ func (dv *DefaultBlockValidator) ValidateSemantic(ctx context.Context, child *ty
 
 // ValidateSyntax validates a single block is correctly formed.
 func (dv *DefaultBlockValidator) ValidateSyntax(ctx context.Context, blk *types.Block) error {
+	// TODO special handling for genesis block
+	// figure out in: https://github.com/filecoin-project/go-filecoin/issues/3121
+	if blk.Height == 0 {
```

Comment:
Currently the genesis blocks we create fail validation, this check was added as a temporary fix to avoid running validation on genesis blocks.

Reply:
An alternative I'd prefer slightly to this is to make the validator aware of the genesis cid (pass it in during construction the same way expected gets it).  This way we are skipping only genesis.  Fine to leave that for #3121.